index.bs

```diff
@@ -282,30 +266,164 @@ Loader instances are initially created with the internal slots described in the
   <tr>
     <td>\[[Registry]]</td>
     <td>An object</td>
-    <td>An instance of Registry (<a href="#registry">4</a>).</td>
+    <td>An instance of <a href="#registry">Registry</a>.</td>
   </tr>
 </table>
 
 <h2 id="registry">Registry Objects</h2>
 
-<h3 id="registry-abstract-operations">Abstract Operations for Registry Objects</h3>
+<h3 id="registry-constructor">The Registry Constructor</h3>
+
+The Registry constructor is the %Registry% intrinsic object. When called as a constructor it creates and initializes a new %Registry% object. %Registry% is not intended to be called as a function and will throw an exception when called in that manner.
```

Comment:
Agreed, the loader sets the [[Registry]] internal slot automatically, whether that is an internal %Registry% or just a Map.